content/blog/2021-03-03-dvc-2-0-release.md

```diff
@@ -142,8 +142,7 @@ Experiment results have been applied to your workspace.
 
 In the examples above, hyperparameters were changed with the `--set-param`
 option, but you can make these changes by modifying the params file instead. In
-fact _any code or data files can be changed_ and `dvc exp run` will capture the
-variations.
+fact _any code can be changed_ and `dvc exp run` will capture the variations.
```

Comment:
This is the only substantive change I made. Data dependencies are always reset to the last git-committted version in dvc.lock right now. This is needed because the current implementation of checkpoints rolls back to the last committed version. We may revisit this.